code/game/mecha/mecha.dm

```diff
@@ -614,21 +614,14 @@
 	else if(hasInternalDamage(MECHA_INT_CONTROL_LOST))
 		set_glide_size(DELAY2GLIDESIZE(step_in))
 		move_result = mechsteprand(movemode)
-		if (occupant)
```

Comment:
Dont remove these, they are vital to how mob movement works.

Reply:
It worked fine in testing.

Reply:
As in, I removed them, and I could move around in the mech, including entering & exiting, just fine.

Reply:
This has nothing to do with the mechs speed your breaking the occupants speed timers. These are needed

Reply:
Wouldn't the timers fix themselves as soon as the user move outside the mech anyway?
From my experience with other code stuff, those kind of timer just check if if it happened at least X time ago, and doesn't actually care if it is greater than the threshold.
And if it does break something, we can just reset it when we leave the mech, instead of every time we move.

Reply:
We need it in move as thats when you move.
In cases were you do not exit the mech can lead to edge issues that this fixes
These lines are required, no point in cutting your own finger to put a bandaid fix on it.

Reply:
Yeah, but technically, you aren't moving, it is the mech that move, and the mech has its own limitation on how fast it can move.